third_party/icu/source/common/putil.cpp

```diff
@@ -62,7 +62,7 @@
  */
 #include <time.h>
 
-#if !U_PLATFORM_USES_ONLY_WIN32_APII && !defined(STARBOARD)
```

Comment:
Interesting how Change If056fb1c replaced
`U_PLATFORM_USES_ONLY_WIN32_API` with
`U_PLATFORM_USES_ONLY_WIN32_APII`.
I recall noticing that in #1474.